docs/content_management/locations.md

```diff
@@ -94,10 +94,12 @@ When the Location of a Content item is hidden, the system will not display it on
 
 !!! caution "Visibility and permissions"
 
-    The Location visibility flag is not permission-based and thus acts as a simple potential filter.
-    **It is not meant to restrict access to content**.
+    The visibility flag is a convenient feature for withdrawing content from the frontend. It acts as a simple filter in the default frontend templates.
```

Comment:
```suggestion
    The visibility flag is a convenient feature for withdrawing content from the frontend. It acts as a filter in the frontend by default, but in your code you can choose to respect it or ignore it.
```
@MagdalenaZuba New proposal. Uses "you", makes it clear we're talking about development, not end-user actions, and doesn't specifically mention templates, as the code can be wherever.

Reply:
> @MagdalenaZuba New proposal. Uses "you", makes it clear we're talking about development, not end-user actions, and doesn't specifically mention templates, as the code can be wherever.

```suggestion
    The visibility flag is a convenient feature for withdrawing content from the frontend. It acts as a filter in the frontend by default. You can choose to respect it or ignore it in your code .
```

Reply:
Thanks, that's better!